examples/checkbox/checkbox-2/checkbox-2.html

```diff
@@ -72,12 +72,12 @@ <h2 id="ex_label">Example</h2>
             window.onload = function () {
               var i;
               var mixed = document.querySelectorAll('[role="checkbox"]');
-              for (i = 0; i < mixed.length; i++) {
+              for (var i = 0; i < mixed.length; i++) {
                 var m = new CheckboxMixed(mixed[i]);
                 m.init();
               }
               var checkboxes = document.querySelectorAll('input[type="checkbox"]');
-              for (i = 0; i < checkboxes.length; i++) {
+              for (var i = 0; i < checkboxes.length; i++) {
```

Comment:
`i` is declared at the top of the function. I think this should be reverted.

Reply:
Done